src/fu-tool.c

```diff
@@ -3132,6 +3132,10 @@ fu_util_version(FuUtilPrivate *priv, GError **error)
 	g_autoptr(GHashTable) metadata = NULL;
 	g_autofree gchar *str = NULL;
 
+	/* load engine */
```

Comment:
This code change no longer needed if not looking at compile version right?

Reply:
Nah, we do need to init the plugins to get the runtime version. Thanks for all the reviews btw.